nowcasting_forecast/config/power_perceiver.yaml

```diff
@@ -0,0 +1,61 @@
+general:
+  description: Config for producing batches on for live forecasts.
+  name: power_perceiver
+input_data:
+  #---------------------- GSP -------------------
+  gsp:
+    gsp_zarr_path: ./nowcasting_forecast
+    history_minutes: 60
+    forecast_minutes: 480
+    log_level: "INFO"
+    is_live: True
+
+  #---------------------- NWP -------------------
+  nwp:
+    nwp_channels:
+      - t
```

Comment:
We only have four channels in live NWP data. These are
- dswrf
- si10
- t
- rprate